src/Microsoft.ML.Data/DataLoadSave/Database/DatabaseSource.cs

```diff
@@ -25,9 +25,29 @@ public DatabaseSource(DbProviderFactory providerFactory, string connectionString
             CommandText = commandText;
         }
 
+        /// <summary>Creates a new instance of the <see cref="DatabaseSource" /> class.</summary>
+        /// <param name="providerFactory">The factory used to create the <see cref="DbConnection"/>..</param>
```

Comment:
Why the double ending `..` ?

Reply:
changing here and in the above constructor.

---
In reply to: [355678098](https://github.com/dotnet/machinelearning/pull/4494#discussion_r355678098) [](ancestors = 355678098)